plans/example.fmf

```diff
@@ -0,0 +1,6 @@
+summary: Test the evaluation of llvm_sb macro
+discover:
+    how: shell
+    tests:
+    - name: test evaluation of llvm_sb macro
+      test: rpm --eval "%{llvm_sb}" | grep "llvm_snapshot_version:"
```

Comment:
Just to be clear, the `execute` should be defined on the top-level (i.e. on the same hierarchy level as `discover`)

Reply:
You can also try it locally using `tmt`. E.g. just using `tmt` will list available tests and plans. You can also do `tmt plan show`. You can also run the test using `tmt run`, however based on the test, it looks like the test depends on artifact installation from copr build created by packit so it will fail locally.

Reply:
@FrNecas thank you. I've incorporated your help.